apollo-ios-codegen/Sources/IR/IR+RootFieldBuilder.swift

```diff
@@ -353,6 +383,29 @@ class RootFieldBuilder {
     return InlineFragmentSpread(selectionSet: irSelectionSet)
   }
 
+  private func buildInlineFragmentSpread(
```

Comment:
This is the new function that takes the deferred inline fragment and wraps it in the parent type case, effectively creating a nested type case fragment.